capstone-app/src/app/modules/photos/components/image-container/image-container.component.scss

```diff
@@ -125,4 +125,80 @@ img {
 
 a {
   color: inherit;
+  text-decoration: none;
+}
+
+@media only screen and (max-width: 1200px) {
+  .flex-container {
+    .picture {
+      height: 200px;
+    }
+  }
+}
+
+@media only screen and (max-width: 750px) {
+  .flex-container {
+    .picture {
+      height: 150px;
+    }
+  }
+
+  .click-off {
+    pointer-events: none;
+  }
+  
+  .click-on {
+    pointer-events: auto;
+  }
+}
+
+@media only screen and (max-width: 600px) {
+  .title a {
+    font-size: 14px;
+    font-weight: 200;
+  }
+
+  .flex-container .picture .details .title {
+    line-height: 0.75;
+  }
+
+  .flex-container .picture .details span {
+    bottom: -30px;
+    padding: 20px 5px;
+  }
+
+  .picture:hover, .picture:active {
+    .details {
+      span {
+        bottom: -20px !important;
```

Comment:
My three least favorite css hacks in one line. Important, negatives, and relative/absolute positioning. Anything we can do about this?